src/components/InteractiveArticle.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+import Helmet from 'react-helmet';
+import NotFound from 'components/NotFound';
+
+export default class InteractiveArticle extends FalcorController {
+  static getFalcorPathSets(params) {
+    return [
+      // For now we only use the html part, the js and css parts are for further improvements
+      ['interactiveArticlesBySlug', params.articleSlug, ['html', 'js', 'css']],
+      // Fetch article data
+      [
+        'interactiveArticlesBySlug',
+        params.articleSlug,
+        ['title', 'teaser', 'html', 'published_at', 'issueNumber', 'category', 'slug', 'image'],
+      ],
+      ['interactiveArticlesBySlug', params.articleSlug, 'authors', { length: 10 }, ['name', 'slug']],
+
+    ];
+  }
+
+  render() {
+    if (this.state.ready) {
+      if (!this.state.data) {
+        return <NotFound />;
+      }
+      const articleSlug = this.props.params.articleSlug;
+      // Access data fetched via Falcor
+      const articleData = this.state.data.articlesBySlug[articleSlug];
+      // make sure article meta image has default
+      const articleMetaImage = articleData.image || 'https://thegazelle.s3.amazonaws.com/gazelle/2016/02/saadiyat-reflection.jpg';
+      const meta = [
+        // Search results
+        { name: 'description', content: this.props.teaser },
```

Comment:
What is this? And there is no `teaser` prop here, we agree on that right?

Reply:
@zanemountcastle and if you could just reply to this as well :)

Reply:
Oh, this is supposed to be `articleData.teaser`

Reply:
Speaking of, it doesn't look like `articleData` will work with how falcor is

Reply:
I think with the way I changed the falcorPath it should work. But yeah good, thanks for the quick reply :).

Reply:
I'm fixing this in my rebase that will be force pushed in the near future